### PR TITLE
Feature/owldrop 2

### PR DESCRIPTION
**Pull request by Velenir**
Adds multiplier to reflect 

> OWL Tokens shall be generated at a rate of twoten (2 ~10~) OWL Tokens for every one (1) GNO Token

and make Airdrop more flexible

### CODE REVIEW

File: migrations/4_deploy_OWL_airdrop.js [hunk 1]
```diff
@@ -16,6 +22,7 @@ module.exports = function (deployer, network, accounts) {
     network,
     accounts,
     web3,
-    gnoLockEndTime
+    gnoLockEndTime,
+    multiplier: process.env.MULT
```
Comment (anxolin) [inline migrations/4_deploy_OWL_airdrop.js]: I would set a name more semantic, like OWL_PER_GNO, also, I think is best if we assert that it was provided.

File: src/migrations-truffle-4/4_deploy_OWL_airdrop.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 function migrate ({
   artifacts,
   deployer,
   network,
   accounts,
   web3,
-  gnoLockEndTime = _getDefaultLockEndTime()
+  gnoLockEndTime = _getDefaultLockEndTime(),
+  multiplier = OWL_MULTIPLIER
```
Comment (anxolin) [inline src/migrations-truffle-4/4_deploy_OWL_airdrop.js]: same here

File: src/migrations-truffle-5/4_deploy_OWL_airdrop.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 
-async function migrate({
+async function migrate ({
   artifacts,
   deployer,
   network,
   web3,
-  gnoLockEndTime = _getDefaultLockEndTime()
+  gnoLockEndTime = _getDefaultLockEndTime(),
+  multiplier = OWL_MULTIPLIER
```
Comment (anxolin) [inline src/migrations-truffle-5/4_deploy_OWL_airdrop.js]: ...and here :)

**Comment by Velenir** [comment]
Can deploy the next airdrop with command:
```
PK=<TokenOWL creator> OWL_PER_GNO=2 LOCK_END_TIME='2019-05-05T12:00:00+02:00' npx truffle migrate --network mainnet --reset
```

**Pull request merged**